index.js

```diff
@@ -49,6 +53,9 @@ module.exports = {
   format: function(text, opts) {
     return formatWithShebang(text, normalizeOptions(opts));
   },
+  formatAST: function(ast, opts) {
```

Comment:
Pushed this fix, I figure you mean that the acceptable AST format might change to support e.g. flow-parser as well as babylon.